lintconfig_base.json

```diff
@@ -9,7 +9,7 @@
     "test": true,
     "sort": ["path"],
     "Linters": {
-        "goimports": {"Command": "goimports -l --local istio.io"}
```

Comment:
@mandarjog I have problem changing this. The linter stops to work deterministically.

Reply:
`bin/fmt.sh` does not work ?

Reply:
Found the problem. I should create a check only model for fmt.sh.